pkg/ddl/syncer/syncer.go

```diff
@@ -126,16 +128,98 @@ type SchemaSyncer interface {
 	// the latest schema version. (exclude the isolated TiDB)
 	// It returns until all servers' versions are equal to the latest version.
 	OwnerCheckAllVersions(ctx context.Context, jobID int64, latestVer int64) error
+	// SyncJobSchemaVerLoop syncs the schema versions on all TiDB nodes for DDL jobs.
+	SyncJobSchemaVerLoop(ctx context.Context)
```

Comment:
can we move it inside NewSchemaSyncer?

Reply:
it's part of DDL, start background routines in `ddl.Start` should be more clear.